File: webapi/FIT_Api_Examples/Modul3_MaticnaKnjiga/ViewModels/MaticnaKnjigaVM.cs
using FIT_Api_Examples.Modul3_MaticnaKnjiga.Models;
using System.Collections.Generic;

namespace FIT_Api_Examples.Modul3_MaticnaKnjiga.ViewModels
{
    public class MaticnaKnjigaVM
    {
        public int id { get; set; }
        public string ime { get; set; }
        public string prezime { get; set; }
        public List<ListaUpisi> listaUpisi { get; set; }
        public float cijenaSkolarina { get; set; }
    }
}

File: arhiva_V7-V10/angular_app/src/app/student-maticnaknjiga/maticna-knjiga-vm.ts
export class MaticnaKnjigaVM {
  id: number
  ime: string
  prezime: string
  listaUpisi: ListaUpisi[]
  cijenaSkolarina: number
}

export class ListaUpisi {
  id: number
  godinaStudija: number
  jelObnova: boolean
  datumUpisZimski: string
  cijenaSkolarine: number
  datumOvjeraZimski: string
  evidentirao_korisnik: string
  akademska_godina_opis: string
  akademska_godina_id: number
}
